sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Operations/ServiceRestClient.cs

```diff
@@ -1065,7 +1074,7 @@ public Response<AnalyzeOperationResult_internal> GetAnalyzeLayoutResult(Guid res
             }
         }
 
-        internal HttpMessage CreateGetCustomModelsNextPageRequest(string nextLink)
+        internal HttpMessage CreateGetCustomModelsNextPageRequest(string nextLink, GetModelOptions? op)
```

Comment:
Any reason not to call op `options`?  It could be short for Operation as well, and it took me a minute to figure it out.